packages/frontend-2/components/settings/workspaces/members/UpdateActionsMenu.vue

```diff
@@ -0,0 +1,194 @@
+<template>
+  <div>
+    <LayoutMenu
+      v-if="filteredActionsItems.length"
+      v-model:open="showMenu"
+      :items="filteredActionsItems"
+      mount-menu-on-body
+      size="lg"
+      :menu-position="HorizontalDirection.Left"
+      @chosen="({ item: actionItem }) => onActionChosen(actionItem)"
+    >
+      <FormButton
+        :color="showMenu ? 'outline' : 'subtle'"
+        hide-text
+        :icon-right="showMenu ? XMarkIcon : EllipsisHorizontalIcon"
+        @click="toggleMenu"
+      />
+    </LayoutMenu>
+
+    <SettingsWorkspacesMembersUpdateDialog
```

Comment:
We will probably need to do this in a more composable way, based on the new design from Michal, but let's leave it like this for now :)